Automattic/jetpack
# Tiled Gallery block: Add Color and Spacing design tools

The proposed change adds background color and spacing (padding and margin) tools to the Tiled Gallery block.

 Before | After |
| ------------- | ------------- |
| ![Markup on 2022-02-10 at 16:32:44](https://user-images.githubusercontent.com/25105483/153440990-7e145e04-db97-49bc-915f-742f620b1080.png) | ![Markup on 2022-02-10 at 16:31:25](https://user-images.githubusercontent.com/25105483/153441048-3b124606-df61-4f6d-bba6-d3fa47cc7f90.png) |

#### Changes proposed in this Pull Request:
* add necessary `supports` (color and spacing)
* make sure the `style` prop is applied to the wrapper `div` of the `Layout` component

#### Jetpack product discussion
This PR is a part of the _Expanding Jetpack Block design tooling for Full Site Editing_ project: pdDOJh-1P-p2 & pdDOJh-6-p2.

#### Does this pull request change what data or activity we track or use?
No, it does not.

#### Testing instructions:
1. Create a new post or a page with the Tiled Gallery block.
2. The `Color` and `Spacing` (displayed as "Dimensions" in the block sidebar) tools should be available
3. Experiment with the tools and observe whether the change applies correctly in the back end
4. Save the changes and review them in the front end as well
5. We can test the change with multiple themes (block-based, classic). Please note that not all themes support the `Color` and `Spacing` tools. The `Geologist` theme supports both tools.
